langfuse/utils/__init__.py

```diff
@@ -67,8 +67,10 @@ def _convert_usage_input(usage: typing.Union[pydantic.BaseModel, ModelUsage]):
         for k in (
             "promptTokens",
             "prompt_tokens",
+            "input_token_count",
             "completionTokens",
             "completion_tokens",
+            "generated_token_count",
```

Comment:
Can we revert this change and handle this in the Langchain callback handler?

- Functionality in here should be abstracted for Langfuse. Hence, everything going in here should already have been converted to prompt_tokens, completion_tokens etc.

How does this work in the langchain case for IBM? We parse the usage from all the different models [here](https://github.com/langfuse/langfuse-python/blob/4d7112a4b293bff5a329a4351abfefbc468d87b0/langfuse/callback/langchain.py#L731). Could you add your logic to `_parse_usage`?

Thanks

Reply:
sure I'll try to make the change

Reply:
usage for us looks like - 
![image](https://github.com/user-attachments/assets/3130b1b8-cb1f-441a-9dc0-923eeb967800)

Reply:
@maxdeichmann I fixed and pushed